build-requirements.txt

```diff
@@ -9,7 +9,7 @@ pathlib2==2.3.2 --hash=sha256:460e67b14d0574b0529a0017b1eb05d10d9722681e303fec70
 python-dateutil==2.7.5 --hash=sha256:f6eb9c17acd5a6954e1a5f2f999a41de3e7e25b6bc41baf6344bd053ec25ceeb
 python-editor==1.0.3 --hash=sha256:e47dcec4ea883853b8196fbd425b875d7ec791d4ede2e20cfc70b9a25365c65b
 requests==2.20.0 --hash=sha256:d87b2085783d31d874ac7bc62660e287932aaee7059e80b41b76462eb18d35cc
-securedrop-sdk==0.0.13 --hash=sha256:c8d98208fb2074336c06be3fef0994a8a57fde7a765cead12bc36e9128d319e2
+securedrop-sdk==0.1.0 --hash=sha256:488417f9f08e4c432c81348dfbd5da0e756ded1737ba58b2ffc8f0e703abc1cb
```

Comment:
matches shasum of whell on our pypi mirror https://pypi.securedrop.org/localwheels/securedrop_sdk-0.1.0-py3-none-any.whl